index.ts

```diff
@@ -230,10 +226,36 @@ export type Stable64GrowResult = Variant<{
     err: StableMemoryError;
 }>;
 
+/**
+ * A decorator for annotating query methods on external canisters. Can only be
+ * used on class properties with a return type of (args: any[]) =>
+ * CanisterResult<T>.
+ *
+ * @example
+ * ```ts
+ * export class SomeOtherCanister extends ExternalCanister {
+ *   @query
+ *   someCanisterMethod: (someParam: SomeParamType) => CanisterResult<SomeReturnType>;
```

Comment:
Method name should be camel case

Reply:
meaning snake case... yes, I can change that. Will change.